spec/controllers/annotation_categories_controller_spec.rb

```diff
@@ -33,6 +34,16 @@
 
       expect(assignment.annotation_categories.count).to eq 1
     end
+
+    it 'successfully creates annotation_category with nil flexible_criterion' do
+      category = create(:annotation_category, assignment: assignment)
```

Comment:
I don't think you want to do this here. Category names need to be unique for an assignment, so I suspect the `post` call below is not actually creating a valid object.

Instead, you can remove this and pass in a hard-coded string (e.g., `'Category 1'`) to the `post`.